playbooks/researchdata.yml

```
@@ -1,12 +1,14 @@
 ---
-- hosts: researchdata_prod
+# by default this playbook runs in the staging environment
+# to run in production, pass '-e runtime_env=production'
+- name: build the research data site
+  hosts: researchdata_{{ runtime_env | default('staging') }}
   remote_user: pulsys
   become: true
   vars_files:
-    - ../group_vars/researchdata/researchdata_prod.yml
+    - ../group_vars/researchdata/staging.yml
```

I'd match the usual pattern, so vars get added in both environments:
```suggestion
    - ../group_vars/researchdata/{{ runtime_env | default('staging') }}.yml
```